File: compiler-group-traslator/src/test/resources/arrayInObjectAssign.java
public class arrayInObjectAssign
{
	public static void main(String[] args)
	{
		Bar1 b;
		int z;
		int[] y;
		
		
		y = new int[]{7,8,9,10};
		b = new Bar1();
		
		b.f = y;
	}
}

class Bar1{
	public int[] f;
}

File: compiler-group-traslator/src/test/resources/c/arrayInObjectAssign.c
#include<stdio.h>
#include<stdlib.h>
#include<stdbool.h>
#include "GC.c"
#include "ExceptionHandlingFunctions.c"
VM* vm;
typedef struct Bar1 {
	int* f;
}Bar1;

int main(char* args) {
	vm = newVM();
	struct Bar1* b;
	int z;
	int* y;

	y = newArray(vm,"int",&y , 5 * sizeof (int), makeList(vm,&y, 5,7, 8, 9, 10 )) ;

	b = (Bar1*)newObject(vm, &b, sizeof (Bar1)) ;

	b->f = y ;
	return 0;
}